### DISCUSSION TITLE
@types/chrome - Incomplete types for 'ManifestPermissions'

### DISCUSSION THREAD

**Discussion by paularah** [DefinitelyTyped/DefinitelyTyped#65233] [Issues with a @types package]
As per the Chrome extension docs, you should be able to replicate the manifest config below. But the current typing do not allow for that. 

Reference: https://developer.chrome.com/docs/extensions/reference/webRequest/

```json
{
  "name": "My extension",
  ...
  "permissions": [
    "webRequest",
    "*://*.google.com/"
  ],
  ...
}
```

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
If I understand correctly, the fields `version` and `manifest_version` are always required; they probably weren't included in the example you linked because it was only meant to demonstrate the `webRequest` permission.

<https://developer.chrome.com/docs/extensions/mv3/manifest/#overview>
